nuitka/build/Onefile.scons

```diff
@@ -688,6 +688,9 @@ target = env.Program(result_exe, source_files + source_targets)
 # Avoid dependency on MinGW libraries.
 if win_target and gcc_mode and not clang_mode:
     env.Append(LINKFLAGS=["-static-libgcc"])
+# on Windows link against imagehlp to support signed applications detection
+if win_target:
```

Comment:
Do you think it would be wrong for Nuitka to call the signing tool itself, or to be informed it will. Because the code you are adding to onefile bootstrap is not conditional, so Nuitka would have to know ahead of time and not include overhead for those not using it.

Reply:
I don't think Nuitka should call the signing tool as signing could take place on another machine (e.g. if using a hw token). I'm fine with adding a option to Nuitka to enable/disable the PE header parsing but I think it should be enabled by default as in morst cases, singing an exe just works.